Modules/_abc.c

```diff
@@ -0,0 +1,818 @@
+/* ABCMeta implementation */
+
+/* TODO: Test this hard in multithreaded context. */
+/* TODO: Think about inlining some calls and/or using macros */
+/* TODO: Use separate branches with "fast paths" */
```

Comment:
Please strip TODO comments before merging.  Usually such comments stay unresolved for years and aren't useful in any way.